### ISSUE TITLE
Problems with the Timbre library

### ISSUE THREAD

**Issue by AndreaCrotti** [cloverage/cloverage#238]
I was trying to add cloverage to a project using timbre, and I can't get it to work.
I had already reported the error on:
https://github.com/ptaoussanis/timbre/issues/274

I'm not sure where the issue belong, but are cljx files supported at all in theory?
If not I guess there isn't much to do, but maybe there are some other ways to get it working?

**Comment by lvh** [comment]
cljc should work and cljx won't :)

**Comment by AndreaCrotti** [comment]
Yeah ok then no hope until we get a newer version of Timbre sadly

**Comment by lvh** [comment]
Yep. I'm honestly even on the fence if going back and supporting cljx is worth it. It's hard enough to upgrade cloverage right now and more code for supporting old & crufty stuff is scary :)

I guess I'll close this issue for now.

**Comment by AndreaCrotti** [comment]
Sure makes sense

**Comment by AndreaCrotti** [comment]
Uhm that's weird though on one of my personal projects I also use timbre and cloverage works
https://codecov.io/gh/AndreaCrotti/elo
so it doesn't "always* blow up https://github.com/AndreaCrotti/elo/blob/master/project.clj#L64

Maybe it's just somehow not loading that same problematic file while running my project.

**Comment by rmangi** [comment]
I'm actually having this same issue and can't seem to resolve it.

**Comment by lvh** [comment]
@rmangi Can you recreate a minimal reproducer?

**Comment by rmangi** [comment]
Hi, thanks for the quick response. So I've isolated the issue to another dependency that uses nippy. It's actually a library that I've contributed to https://github.com/tolitius/cbass so I'll try to track it down there. If you simply add that to a project and run cloverage it fails with "reader-error does not exist". Deep in the stack trace it looks like nippy is loading taoensso.encore and that's where it says the root of the error is (or at least that's what it looks like to me). Googling led me to this issue.

**Comment by rmangi** [comment]
I think I figured it out. That project has a "test" that's actually trying to connect to a real hbase cluster. I mocked it out and pushed a local snapshot and it seems to fix the problem. I assume cloverage is loading in the tests for that dependency and somehow that's causing it to fail. nippy is just a red herring.

Sorry for the alarm :)
